# 椭圆障碍物数组消息定义

# 消息头
std_msgs/Header header

# 障碍物数组
EllipseObstacle[] obstacles

